app.js

```diff
@@ -34,23 +36,63 @@ mongoose.connect("mongodb://localhost:27017/userDB", {
 });
 const userSchema = new mongoose.Schema({
   email: String,
-  password: String
+  password: String,
```

Comment:
Isnt it possible to just add comma (,) rather than removing and readding the stuff?

Reply:
or is it your IDE?

Reply:
my bad, its GH who is making this up.. harder to review